plugins/google-sheets/src/components/CheckboxTextField.tsx

```diff
@@ -27,7 +29,7 @@ export function CheckboxTextfield({ value, darken, checked, onChange }: Props) {
                 checked={checked}
                 onChange={handleCheckboxChange}
                 onClick={e => e.stopPropagation()}
-                className="checked:!bg-sheets-green focus:ring-1 focus:ring-sheets-green"
+                className="checked:!bg-sheets-green focus:ring-1 focus:ring-sheets-green checked:border-none dark:!bg-[#777] !bg-[#CCC]"
```

Comment:
These colors are different because we render this checkbox on a non standard background.